docs/sources/operations/loki-canary.md

```diff
@@ -77,15 +83,16 @@ log lines be sure not to set the two out of order time range flags too far in th
 The defaults are already enough to test this functionality properly, and setting them
 too far in the past can cause issues with the spot check test.
 
-When using `out-of-order-percentage`, you also need to make use of pipeline stages
+
```

Comment:
was there supposed to be an extra newline here?

Reply:
Didn't hurt anything.  But probably no, an extra newline was not needed there.  We can remove it the next time we touch this section.